### ISSUE TITLE
The input for talk to me

### ISSUE THREAD

**Issue by kml74123** [caqlishire/midterm2#2]
I think it will help if you could help if you show us the option for your "talk to me " line because if I enter something not correct to the option you have in the code to use it just keeps repeating " please talk to me" and maybe some maybe not be able to exit the problem or go to next step if they didn't input the right options.  To fix this I think when they enter the wrong option few times, it shows the option, for example, the "hello" and "bye"

**Comment by ShueLee8226** [comment]
I agree with kml74123 and you could try put another input string in there so when the user enter the wrong input and it ask them to enter hello or bye?
